app/components/edit/EditPatientHandout.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+interface PatientHandout {
+  id: string;
+  language: string;
+  link: string;
+  [key: string]: string;
+}
+
+const EditPatientHandout = ({ index, item, handleItemChange }: {
+  index: number;
+  item: patientHandout;
+  handleItemChange: (handoutIndex: number, handout: patientHandout) => null;
```

Comment:
```suggestion
  handleItemChange: (handoutIndex: number, handout: patientHandout) => void;
```

You'll want `void`, not `null`, for functions whose return values you're not supposed to use. A return value of `null` means that the function must return `null`, whereas a return value of `void` is special and means that the function can return anything it wants (including the default return value of `undefined`), but the caller is not allowed to make use of the returned value at all.

Reply:
Thanks for the reminder, I made a mistake there.